packages/block-library/src/social-link/edit.js

```diff
@@ -33,8 +33,8 @@ const SocialLinkEdit = ( { attributes, setAttributes, isSelected } ) => {
 	} );
 
 	// Import icon.
-	const IconComponent = getIconBySite( site );
-	const socialLinkName = getNameBySite( site );
+	const IconComponent = site ? getIconBySite( site ) : null;
+	const socialLinkName = site ? getNameBySite( site ) : '';
```

Comment:
Should it default to: `_( 'Social Icon' )` or use directly the name of the block?

```suggestion
	const socialLinkName = site ? getNameBySite( site ) : _( 'Social Icon' );
```

Reply:
That makes sense, or `Social Link` for consistency?

Reply:
The title of the block itself is _Social Icon_, so I'll use that for now.